### DISCUSSION TITLE
request-verify-token and forgot-password routes returns null

### DISCUSSION THREAD

**Discussion by rcourivaud** [fastapi-users/fastapi-users#936] [Q&A]
## Describe the bug

I've configured FastApi Users and all working well, but I can't generate reset password token it returns null.

## Configuration
 - Python version : 3.9.7
 - FastAPI version : 0.73.0
 - FastAPI Users version :  9.2.4

**Comment by frankie567** [comment]
Tokens are not returned in the response, they are used in the `on_after_verify` and `on_after_forgot_password` handlers. It's all explained in the docs:

* https://fastapi-users.github.io/fastapi-users/usage/routes/#post-request-verify-token
* https://fastapi-users.github.io/fastapi-users/usage/routes/#post-forgot-password
